docs/governance/templates/release_issue.md

```diff
@@ -0,0 +1,54 @@
+# Release {version}
+
+<!--
+This is the release issue template. Make a copy of the markdown in this page
+and copy it into a release issue. Fill in relevent values, found inside {}
+!-->
+Complete Milestone
+------------------
+
+- [ ] Create the next version milestone, use [semantic versioning](https://semver.org/) when naming it to be consistent with the community.
+- [ ] Visit the [milestone](https://github.com/GoogleCloudPlatform/open-match/milestone).
+  - [ ] Review closed issues have appropriate tags.
```

Comment:
These "review" should be reordered and rephrased.  "Review" doesn't say what the review should do.  After a few readings I got it, but I think it could be clearer.  Also include search links.

Example rephrasing:  
-  [  ]  Add the milestone for the release to all [merged pull requests first included in this release.](merged-missing-milestone)

[merged-missing-milestone]: https://github.com/GoogleCloudPlatform/open-match/pulls?utf8=%E2%9C%93&q=is%3Aclosed+no%3Amilestone+is%3Amerged+is%3Apr+

Suggested ordering: Apply the milestone to PRs and Issues first, then search using that milestone to apply the proper tags.